exercises/K-functions-parameters/exercise2.js

```diff
@@ -1,5 +1,9 @@
 // Declare your function first
 
+function divide(a , b) {
```

Comment:
Observe your spaces between parameters (a, b) in order to keep your parameters neat.